dashboard/lib/controllers/run.js

```diff
@@ -0,0 +1,67 @@
+const Run = require('../models/run');
+const db = require('../store');
+const { asyncWrapper } = require('../utils');
+
+const runs = db.getTable('runs');
+
+const api = {
+    insert: asyncWrapper(async (data) => {
+        await runs.insert(data.id, new Run(data));
+    }),
+
+    findOne: asyncWrapper(async (id) => {
+        return id && (await runs.findOne(id));
+    }),
+
+    find: asyncWrapper(async () => {
+        return await runs.find();
+    }),
+
+    clear: asyncWrapper(async () => {
+        return await runs.clear();
+    }),
+
+    pauseRun: asyncWrapper(async (id) => {
```

Comment:
Don't need to add "run" to the name:
```suggestion
    pause: asyncWrapper(async (id) => {
```